eventarc/workflows/main.tf

```diff
@@ -83,7 +83,7 @@ resource "google_workflows_workflow" "workflows_example" {
   provider        = google-beta
   region          = "us-central1"
   description     = "A sample workflow"
-  service_account = google_service_account.eventarc_workflows_service_account.id
+  service_account = google_service_account.eventarc_workflows_service_account.email
```

Comment:
In the official documentation @ https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/workflows_workflow I see `ID` are used. Can you help me understand, why change it to `email`?